CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Upcoming version 5.x.x (`develop` branch)
 
+- [#652]
+  - **Description:** Remove Internet Explorer 11 specific logic from KTextTruncator.
+  - **Products impact:** bugfix.
+  - **Addresses:**  https://github.com/learningequality/kolibri-design-system/issues/643
+  - **Components:** KTextTruncator
+  - **Breaking:** no
+  - **Impacts a11y:** -
+  - **Guidance:** Consumers need to ensure the modal  is still working correctly.
```

Comment:
I don't think these changes have anything to do with the modal, possibly result of copy-paste? Please cleanup.